### ISSUE TITLE
AMD 6900 xt CL_build_program_failure when calling clbuildprogram

### ISSUE THREAD

**Issue by chimik8** [andru-kun/wildrig-multi#67]
CL_BUILD_PROGRAM_FAILURE when calling clBuildProgram
CL_INVALID_KERNEL_NAME when calling clCreateKernel for kernel 'blake_512_80'
This is occuring on VeriBlock lite vprogpov/

**Comment by Weaksfx** [comment]
![image](https://user-images.githubusercontent.com/43636645/135301512-7e834e65-b9c5-4bca-9903-540bb9a4355f.png)
same here AMD 5500 5700 5600

**Comment by UselessGuru** [comment]
Try this:
https://github.com/andru-kun/wildrig-multi/issues/64#issuecomment-918860911

**Comment by Weaksfx** [comment]
> Try this: [#64 (comment)](https://github.com/andru-kun/wildrig-multi/issues/64#issuecomment-918860911)

there is no file on Mega

**Comment by UselessGuru** [comment]
Ups, then the author has removed the file. I have a copy here:
https://github.com/Minerx117/miners/releases/download/WildRig/wildrig-multi-windows-0.29.1.7z

**Comment by andru-kun** [comment]
> Ups, then the author has removed the file. I have a copy here: https://github.com/Minerx117/miners/releases/download/WildRig/wildrig-multi-windows-0.29.1.7z

that one removed because of new one :) Also he need Linux version

**Comment by UselessGuru** [comment]
> that one removed because of new one :) 
Where is the new one?

**Comment by andru-kun** [comment]
> 
> 
> > that one removed because of new one :)
> 
> Where is the new one?

[comment in #66](https://github.com/andru-kun/wildrig-multi/issues/66#issuecomment-922232610). There us sine fixes to other gpu's. Looks like need to release it without finishing firopow %)

**Comment by UselessGuru** [comment]
Problem with X16RV2 in beta version (comment in #66):

wildrig.exe --algo x16rv2 --api-port=4004 --url=x16rv2.eu-west.nicehash.com:3379 --user=##########################.####### --pass=x --multiple-instance --opencl-threads auto --opencl-launch auto --opencl-platforms=1 --opencl-devices=0

```
 * VERSIONS: WildRig/0.29.1 beta libuv/1.21.0 OpenCL/2.0 MSVC/2015
 * CPU:      Intel(R) Core(TM) i5-8600K CPU @ 3.60GHz
 * ALGO:     x16rv2
 * POOL #1:  x16rv2.eu-west.nicehash.com:3379
 * API PORT: 4004
 * COMMANDS: hashrate, statistics, pause, resume
[08:17:41] donation: 1%
[08:17:41] send-stale: no
[08:17:41] watchdog: disabled
[08:17:41] temp limits: 60C..85C
[08:17:41] initializing GPUs...
[08:17:41] NVIDIA(sm75) not fully optimized yet for chosen algorithm
[08:17:41] GPU #0: NVIDIA GeForce GTX 1660 SUPER [busID: 1] [arch: sm75]
[08:17:41] threads: 1, intensity: 22, worksize: 64/1024, cu: 44, mem: 6144Mb
[08:17:41] use pool x16rv2.eu-west.nicehash.com:3379 172.65.226.105
[08:17:41] Stratum set raw difficulty to 8.0000
[08:17:41] new job from x16rv2.eu-west.nicehash.com:3379 diff 134.22M
[08:17:41] block: 702,996       job target: 0x0000001fffe00000
[08:17:41] hash order:
[08:17:41] Jh5SknHamJh5CubFugShvJh5Jh5GroTigLufSknCubTigKckGroShb
[08:17:41] Error CL_INVALID_ARG_INDEX when calling clSetKernelArg for kernel 'tiger_192_64_fill', argument 3
[08:17:42] Error CL_INVALID_ARG_INDEX when calling clSetKernelArg for kernel 'tiger_192_64_fill', argument 3
[08:17:44] Error CL_INVALID_ARG_INDEX when calling clSetKernelArg for kernel 'tiger_192_64_fill', argument 3
[08:17:45] Error CL_INVALID_ARG_INDEX when calling clSetKernelArg for kernel 'tiger_192_64_fill', argument 3
[08:17:46] Error CL_INVALID_ARG_INDEX when calling clSetKernelArg for kernel 'tiger_192_64_fill', argument 3
[08:17:47] Ctrl+C received
[08:17:47] exiting...
[08:17:47] GPU #0: shutting down...
[08:17:47] Error CL_INVALID_ARG_INDEX when calling clSetKernelArg for kernel 'tiger_192_64_fill', argument 3
```

**Comment by andru-kun** [comment]
> Problem with X16RV2 in beta version (comment in #66):

didn't test that algo on NVIDIA, but thanks for report! There is definitely a bug, from time of first release for nvidia gpu's. Will include a fix for it.

**Comment by andru-kun** [comment]
should be fixed in 0.30.0
